qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -84,6 +87,18 @@ def change_virtio_media(cdrom_virtio):
                                             run_install_cmd,
                                             installer_pkg_check_cmd,
                                             copy_files_params=params)
+        session.close()
```

Comment:
I don't know if it is a must to close this session? Is it possible to work session_serial together?

Reply:
Good point.
IMO, if we don't use the session anymore, we can close it and reopen it when needed.

Reply:
I also not sure what is the best practice for that, let us ask for some suggestions from @vivianQizhu

Reply:
> Good point. IMO, if we don't use the session anymore, we can close it and reopen it when needed.
It will open a session file log when starting a new session, currently, we have many many session logs already.

Reply:
@leidwang I agree with @xiagao As you only want to start another serial session, close and reopen current ssh session will only create more log files and mess debugging.
Besides, I would recommend move the block from line 129 directly above to up 101, another check for same condition is unnecessary.

Reply:
> if we don't use the session anymore

@leidwang I understand that this recommendation itself is good, but it says 'if we don't use the session anymore', this case we actually still use it in the following steps.

Reply:
Okay, I will update the code.Thanks @xiagao @vivianQizhu

Reply:
> @leidwang I agree with @xiagao As you only want to start another serial session, close and reopen current ssh session will only create more log files and mess debugging. Besides, I would recommend move the block from line 129 directly above to up 101, another check for same condition is unnecessary.

Hi @vivianQizhu 

Line 129 is used to check the static network configuration after upgrading the driver through the installer. We need check it after upgrade driver via installer, so we have to check it twice,  the first time after we set it up and the second time after upgrading the driver.

Reply:
@leidwang Right, I missed some lines between them.